sdk/dotnet/Image.cs

```diff
@@ -129,6 +129,8 @@ public Image(string name, ImageArgs args, ComponentResourceOptions? options = nu
             {
                 var imageName = imageArgs.Item1;
 
+                Docker.RunCommandThatMustSucceed("docker", new[] { "ps"}, this).ConfigureAwait(false);
```

Comment:
Why will this run fundamentally earlier than `BuildAndPushImage` several lines down below?